WordPress/Classes/Utility/Blogging Reminders/BloggingRemindersScheduler.swift

```diff
@@ -112,11 +112,11 @@ class BloggingRemindersScheduler {
     private static var defaultDataFileName = "BloggingReminders.plist"
 
     private static func defaultDataFileURL() throws -> URL {
-        guard let url = FileManager.default.containerURL(forSecurityApplicationGroupIdentifier: WPAppGroupName) else {
-            throw Error.cantRetrieveContainerForAppGroup(appGroupName: WPAppGroupName)
-        }
-
-        return url.appendingPathComponent(defaultDataFileName)
+        try FileManager.default.url(for: .documentDirectory,
```

Comment:
I think we should use the `applicationSupportDirectory` directory instead.

From https://developer.apple.com/library/archive/documentation/FileManagement/Conceptual/FileSystemProgrammingGuide/FileSystemOverview/FileSystemOverview.html#//apple_ref/doc/uid/TP40010672-CH2-SW13:

#### About the "Documents" directory:

> Use this directory to store user-generated content. The contents of this directory can be made available to the user through file sharing; therefore, this directory should only contain files that you may wish to expose to the user.

#### About the `ApplicationSupport` directory:

> Use this directory to store all app data files except those associated with the user’s documents. For example, you might use this directory to store app-created data files, configuration files, templates, or other fixed or modifiable resources that are managed by the app. An app might use this directory to store a modifiable copy of resources contained initially in the app’s bundle. A game might use this directory to store new levels purchased by the user and downloaded from a server.

Edit: I think this requires `Application supports iTunes file sharing` to be ON, but IMHO considering the guidance above the Application Support directory seems like the best fit for additional configuration files, IMHO.

Reply:
Good point @diegoreymendez thanks for finding this! I updated to `applicationSupportDirectory`. It did not seem to need the iTunes Sharing when I built locally, but let's make sure we test it on the next beta cut, cc @mokagio